### PR TITLE
Add remove edit

### PR DESCRIPTION
**Pull request by jerryowusu**
## Add-Remove-Edit

In this milestone I:

- Created functions to add and remove task lists from a dynamic array.
- Implemented a function to edit tasks descriptions.
- All tasks deleted also updated the remaining item's indexes.
- All changes on the To-Do lists are also updated on local storage

### CODE REVIEW

**Review by BrianSammit — CHANGES_REQUESTED** [review]
.

File: src/modules/taskClass.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  }
+
+  getStoredTodos() {
+    this.allTodos = JSON.parse(localStorage.getItem('todos'));
+  }
+
+  editTodo(newValue, index) {
+    this.allTodos[index - 1].description = newValue;
+    this.saveTodo();
+  }
```
Comment (BrianSammit) [inline src/modules/taskClass.js]: - [x] When I edit a task and give a space I got a bug when I reload the page. 

Capture

Kindly fix this minor issue. 😄 💪

**Review by BrianSammit — CHANGES_REQUESTED** [review]
# CHANGES REQUIRED 🔴 🛑
Hello @jerryowusu, great job you are doing so far in this project. 💯

TO HIGHLIGHT:

- Your project is professional 
- Your repo is professional 
- Well structured files 
- Great job in adding deleting and editing the task
- The index is properly assigned

However, there are a few changes to make to move to the next project 🛠️ . Keep working hard 💪 💻 

## Required Changes ♻️
See my inline comment to know what changes are needed. 👇 👀 

Please make these required changes and send it again for another review 👍 💪
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
# HAPPY CODING!!! 😄 

cat


**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**

--------

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: src/modules/taskClass.js (hunk 1 above)
Comment (jerryowusu) [reply src/modules/taskClass.js]: Resolved. Thanks

**Review by rustam-tolipov — APPROVED** [review]
## Hi @jerryowusu 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 🎉 🎉 🎉 🎉, 

Cheers and Happy coding!👏👏👏

Developer-512px-10

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**